src/v/utils/tests/fragmented_vector_test.cc

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "random/generators.h"
+#include "utils/fragmented_vector.h"
+
+#include <boost/test/unit_test.hpp>
+
+#include <vector>
+
+template<typename T>
+static void
+test_equal(std::vector<T>& truth, fragmented_vector<T, 1024>& other) {
```

Comment:
nit: that can possibly be replaced with or implemented using `BOOST_REQUIRE_EQUAL_COLLECTIONS`

Reply:
fixed